packages/enzyme/src/ReactWrapper.js

```diff
@@ -841,7 +841,8 @@ class ReactWrapper {
         throw new TypeError('ReactWrapper::invoke() requires the name of a prop whose value is a function');
       }
       return (...args) => {
-        const response = handler(...args);
+        const wrapAct = this[RENDERER].act || function wrapAct(fn) { return fn(); };
```

Comment:
instead of the `||` here, let's either add a default `act` to `EnzymeAdapter`, or, let's just avoid the wrapping where it's not needed.

Reply:
ping @chenesan; this remains

Reply:
Ah, just changed it... Let's revert it back

Reply:
Sorry for misunderstanding. Just changed the logic so it will only wrap `handler` call with `.act` when `this[RENDERER].act` exists.